reading of a file using fgets() .c

```diff
@@ -0,0 +1,17 @@
+#include<stdio.h>
+main()
+{
+  char str[10];
+  file *fp;
```

Comment:
Kindly type the "file" code in capital letters as "FILE *fp"  otherwise this identifier is labelled as undefined in VS Code.